### PR TITLE
Training Bokkens - "funny wood sticks for parrying and beating the shit out of people"

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
Adds training bokkens, shoddily-recolored katanas made of wood that are parrying capable and generally meant for beating the shit out of people.
- 7 (brute) force OR 9 (stam) force, bokken can only deal one damage type at a time
- force can be increased slightly (literally +1 damage) by shoving a metal rod into it
- a successful parry's counterattack has 2.5x damage multiplier (or at least it should)
- you can rename it with a pen
- you can burn some text onto it with a welder
- you can point it at people (combat mode rightclick in case you didn't know idiot)
## Why It's Good For The Game
parrying is fun and maintenance ronin who beat the shit out of cringe people is a funny mental image
## Changelog
:cl:
add: Training bokkens! Make 'em from wood, use 'em in-hand to toggle between harmful and not-so-harmful, practice your parrying with them!
fix: Marker beacons should have a sprite again.
/:cl:

### CODE REVIEW

**Comment by timothyteakettle** [comment]
this is based

**Comment by ghost** [comment]
pr is fine broken shit is on master

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -249,6 +249,78 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+/obj/item/melee/bokken/Initialize()
+	. = ..()
+	AddElement(/datum/element/sword_point)
+
+/obj/item/melee/bokken/attack_self(mob/user)
+	harm = !harm
+	if(harm)
+		damtype = BRUTE
+	else
+		damtype = STAMINA
```
Comment (ghost) [inline code/game/objects/items/weaponry.dm:315]: should do more damage than brute because 140 stam vs 100 brute and also stam regen

**Comment by rossark** [comment]
bokk go bonk

**Comment by Ragolution** [comment]
Why snowflake this and not make it a subtype of baseball bats and/or katanas?

**Comment by Hatterhat** [comment]
spite @Ragolution

**Comment by silicons** [comment]
hey this is kinda based

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -251,45 +251,53 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
 	parry_efficiency_perfect = 120
 	parry_failed_stagger_duration = 3 SECONDS
-	// parry_cooldown = 0.5 SECONDS // most of this items strength *is* in parrying...
 	parry_data = list(PARRY_COUNTERATTACK_MELEE_ATTACK_CHAIN = 2.5) // 7*2.5 = 17.5, 8*2.5 = 20, 9*2.5 = 22.5, 10*2.5 = 25
 
+/datum/block_parry_data/bokken/quick_parry // emphasizing REALLY SHORT PARRIES
+	parry_stamina_cost = 6 // still more costly than most parries, but less than a full bokken parry
+	parry_time_active = 5 // REALLY small parry window
+	parry_time_perfect = 2.5 // however...
+	parry_time_perfect_leeway = 1.25 // the entire time, the parry is perfect
```
Comment (silicons) [inline code/game/objects/items/weaponry.dm]: this should be 2.5

File: code/game/objects/items/weaponry.dm [hunk 3]
```diff
@@ -249,6 +249,118 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+	parry_efficiency_considered_successful = 65		// VERY generous
+	parry_efficiency_perfect = 120
+	parry_failed_stagger_duration = 3 SECONDS
+	parry_data = list(PARRY_COUNTERATTACK_MELEE_ATTACK_CHAIN = 2.5) // 7*2.5 = 17.5, 8*2.5 = 20, 9*2.5 = 22.5, 10*2.5 = 25
+
+/datum/block_parry_data/bokken/quick_parry // emphasizing REALLY SHORT PARRIES
+	parry_stamina_cost = 6 // still more costly than most parries, but less than a full bokken parry
+	parry_time_active = 5 // REALLY small parry window
+	parry_time_perfect = 2.5 // however...
+	parry_time_perfect_leeway = 1.25 // the entire time, the parry is perfect
```
Comment (silicons) [inline code/game/objects/items/weaponry.dm]: this should be 2.5

**Pull request merged**